benchmarks/superphot_plus/test_fit.py

```diff
@@ -0,0 +1,43 @@
+"""Benchmarks the available fitting methods."""
+
+import os
+import pytest
+
+from superphot_plus.ztf_transient_fit import dynesty_single_file
+from superphot_plus.fit_numpyro import numpyro_single_file
+from superphot_plus.import_ztf_from_alerce import (
+    generate_single_flux_file,
+    save_datafile,
+    import_lc,
+)
+
+OUTPUT_DIR = "benchmarks/data/"
+
+test_sn = "ZTF22abvdwik"  # can change to any ZTF supernova
+lc_fn = os.path.join(OUTPUT_DIR, test_sn + ".csv")
+fn_to_fit = os.path.join(OUTPUT_DIR, test_sn + ".npz")
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup():
+    """Generates sample data file for benchmarking purposes."""
+    os.makedirs(OUTPUT_DIR, exist_ok=True)
+    generate_single_flux_file(test_sn, OUTPUT_DIR)
```

Comment:
This should probably just use the data file from /benchmarks/data/. The `generate_single_flux_file` will make a remote call to fetch fresh light curve data, and I think it would be best to continue to use the same CSV (both to avoid the remote call, and to make sure the data stays the same between benchmarking runs).

Reply:
Thanks @delucchi-cmu, it makes sense!